### PR TITLE
0.16 particle protocols.

### PR DESCRIPTION
**Pull request by Muqsit**
12 and 18 were causing client-sided crashes btw..

### CODE REVIEW

File: src/pocketmine/level/particle/Particle.php [hunk 1]
```diff
@@ -24,41 +24,43 @@
 use pocketmine\math\Vector3;
 use pocketmine\network\protocol\DataPacket;
 
-abstract class Particle extends Vector3{
-
+abstract class Particle extends Vector3{	
+
```
Comment (SOF3) [inline src/pocketmine/level/particle/Particle.php]: Why?

Comment (Muqsit) [inline src/pocketmine/level/particle/Particle.php]: same thing right?

**Comment by dktapps** [comment]
We're already testing particles internally and a few of these IDs are incorrect. Thanks anyway.

**Comment by Muqsit** [comment]
no probs mate.

**Pull request closed without merging**